<launch>
    <node pkg="ltu_actor_route_pothole" type="pothole_detection" name="pothole_detection" output="screen">
        <param name="input" value="/cam_repub_blob/image" />
    </node>
</launch>
